### PR TITLE
Day 1 - Initialize project with components

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Micronaut Reviewer 

## Bookstore: initialize project with components

[See the Pull Request Objectives](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md)

### Project requirements

- [x] Initialise React app.
- [x] Add React Redux (npm install react-redux).
- [x] Structure your application files using a "feature folder" approach and use the ducks pattern for your Redux files. 
- [x] The building blocks of your app should be set as re-usable components.
- [x] Add React Router and set two <Route>s and <Link>s for the app's navigation:
  - [x] Books - the default view
    - [x] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet)
    - [x] Should have a form for adding a book (no functionality yet)
  - [x] Categories
    - [x] Should display "Under construction" text only.
- [x] Styling is not required at this point

### Pair Programming 🎳

For this week project, there is no PP required. 

Happy review :)

### CODE REVIEW

**Comment by benjp009** [comment]
Screenshot 2022-01-17 at 14 55 41

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Invalid submission 

Hi,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has node modules in project directory.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by benjp009** [comment]
> # Invalid submission


My bad, problem solved

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link,
+} from 'react-router-dom';
+// eslint-disable-next-line import/extensions
+import Books from './redux/books/Books';
+// eslint-disable-next-line import/extensions
+import Categories from './redux/categories/Categories.js';
+
```
Comment (SajjadAhmad14) [inline src/App.js]: These two imports raise errors and stop the server to run. Please fix the errors in the dev server.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Benjamin, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by elyor-doniyorov — APPROVED** [review]
Hi @benjp009, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Approved
- [x] [OPTIONAL] Please add commands git clone `<project GitHub link>` and cd `<project name>` to `Installation && setup` section of your Readme to make the process more clear for the users.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**